R/module_teal_with_splash.R

```diff
@@ -126,11 +126,15 @@ srv_teal_with_splash <- function(id, data, modules, filter = teal_slices()) {
         )
       )
 
-      validate(need(teal.data::datanames(data), "Data has no datanames. Contact app developer."))
-
-
       is_modules_ok <- check_modules_datanames(modules, teal.data::datanames(data))
-      validate(need(isTRUE(is_modules_ok), is_modules_ok))
+      if (!isTRUE(is_modules_ok)) {
+        showNotification(
+          "Object datanames is not specified; defaults will be applied from the environment",
+          type = "warning",
+          duration = 10
+        )
```

Comment:
This notification will be shown to the app user but is meant for the app developer. The user knows nothing about `datanames`, either from from `teal_data` or the `modules` side.

Reply:
If the app user is not familiar with the datanames, then notifications are unnecessary, I will removed it.